process-engine/skeleton
# Validate XML on Deployment

Depends on:
* https://github.com/process-engine/process_engine/pull/139

## What did you change?

This PR is only for testing purposes and as a marker to update the dependencies after merging.

## How can others test the changes?

* run the `process-engine-server` from this branch
* run the BPMN-Studio from branch `feature/migrate_bpmn_studio_to_new_stack`
* try to add an empty pool to an otherwise working process and deploy it
   * there should appear an error notification (code 422)

## PR-Checklist

Please check the boxes in this list after submitting your PR:
